soccer/gameplay/robocup-py.cpp

```diff
@@ -326,6 +500,26 @@ BOOST_PYTHON_MODULE(robocup)
 	class_<Geometry2d::Circle, bases<Geometry2d::Shape> >("Circle", init<Geometry2d::Point, float>())
 		.def("intersects_line", &Circle_intersects_line)
 		.def("nearest_point", &Geometry2d::Circle::nearestPoint)
+        .def("contains_point", &Geometry2d::Circle::containsPoint)
+        .def("center", &Circle_get_center)
```

Comment:
I think `.def` only works with methods, but there is also `.def_readwrite` and `.def_readonly` that allow you to add accessors for fields.